src/components/Home.tsx

```diff
@@ -4,12 +4,13 @@ import { Button, Grid } from '@material-ui/core';
 import HomeButtonsMajor from './HomeButtonsMajor';
 import HomeButtonsMinor from './HomeButtonsMinor';
 import Search from './Search';
+import { THomeButton } from '../types';
 
-const renderButtons = (buttons, props) =>
-  buttons.map((button, index) => {
+const renderButtons = (buttons: THomeButton[], props: any) =>
```

Comment:
```suggestion
const renderButtons = (buttons: THomeButton[], props: { className: string }) =>
```

Reply:
`any` is 😕

Reply:
how do you feel about defining the props explicitly in an [interface before the component](https://github.com/CodeForBoulder/upswyng/pull/52/commits/7f73281dd333f65fc3fbbfd55e9a7cf09eb6327d)?